feathr_project/feathr/feature_derivations.py

```diff
@@ -16,15 +16,18 @@ class DerivedFeature(FeatureBase):
         key: All features with corresponding keys that this derived feature depends on
         input_features: features that this derived features depends on
         transform: transformation that produces the derived feature value, based on the input_features
+        registry_tags: A dict of (str, str) that you can pass to feature registry for customization. For example, you can use `registry_tags` to indicate feature description, whether this feature is deprecated or not, last refreshed time, etc.
```

Comment:
it's better to have a class for registry_tags so you don't have to keep repeating the documenation. and also can provide some methods for registry tags.